ui/src/entities/User.ts

```diff
@@ -41,3 +45,14 @@ export const fetchCurrentUser = async (): Promise<User> => {
 
   return data.me?.length ? data.me[0] : undefined;
 };
+
+export const sendPasswordResetOTP = async (payload: SendOTPPayload): Promise<undefined> => {
+  const query = `
+    mutation($email: String!) {
+      sendPasswordResetOtp(email: $email)
+    }
+  `;
+  const data = await client.request(query, payload);
+
+  return data ? data.sendPasswordResetOtp : undefined;
```

Comment:
We don't need to check for absence of `data` this way. We can use [optional-chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) feature of typescript.